### PR TITLE
Move nh13 to conda incubator

### PR DESCRIPTION
**Pull request by nh13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -569,24 +568,6 @@ Java SE 8 is required.
 
 `java -jar jars/conda-env-builder.jar` to see the commands supported.  Use `java -jar jars/conda-env-builder.jar <command>` to see the help message for a particular command.
 
-## Include in your project
```
Comment (nh13) [inline README.md:572]: this is confusing, so I removed it

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ jobs:
           bash <(curl -s https://codecov.io/bash) -c -F tools -f '!*.txt'
 
   release-github:
-    if: github.repository == 'nh13/conda-env-builder' && github.ref == 'refs/heads/main'
+    if: github.repository == 'conda-incubator/conda-env-builder' && github.ref == 'refs/heads/main'
```
Comment (nh13) [inline .github/workflows/build.yml:38]: we will need a secret named `REPO_ACCESS_TOKEN` to make this work

File: README.md (hunk 1 above)
Comment (jezdez) [reply README.md:572]: Gotcha!

File: .github/workflows/build.yml (hunk 2 above)
Comment (jezdez) [reply .github/workflows/build.yml:38]: Ah, it's still there FWIW!

Comment (nh13) [reply .github/workflows/build.yml:38]: Phew!

File: build.sc [hunk 3]
```diff
@@ -145,7 +145,7 @@ object tools extends CommonModule {
     ivy"org.slf4j:slf4j-nop:1.7.6"  // For logging silence: https://www.slf4j.org/codes.html#StaticLoggerBinder
   )
 
-  /** Build a JAR file from the com.github.nh13.condaenvbuilder.tools project. */
+  /** Build a JAR file from the com.github.conda-incubator.condaenvbuilder.tools project. */
```
Comment (jezdez) [inline build.sc:148]: I think this should be different:
```suggestion
  /** Build a JAR file from the com.github.condaincubator.condaenvbuilder.tools project. */
```

**Review by jezdez — COMMENTED** [review]
LGTM!

**Review by jezdez — APPROVED** [review]

**Pull request merged**